### PR TITLE
feat: [CI-7469]: Add filenames in xml output for jest tests

### PR DESCRIPTION
**Pull request by anurag-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by anurag-harness** [comment]
retrigger titlecheck

**Comment by anurag-harness** [comment]
retrigger titlecheck

**Comment by anurag-harness** [comment]
retrigger jest

File: package.json [hunk 1]
```diff
@@ -272,5 +272,8 @@
       ]
     },
     "commonChunkLimit": "70-pipeline"
+  },
+  "jest-junit": {
+    "addFileAttribute": "true"
```
Comment (vardanbansal-harness) [inline package.json:277]: Do you have a sample xml with this change enabled?

Comment (anurag-harness) [reply package.json:277]: Yes, ran it locally. This is an example of a testsuite
`  </testsuite>
  <testsuite name="Inline Input Set Error Exp" errors="0" failures="0" skipped="0" timestamp="2023-03-20T07:06:29" time="20.871" tests="18">
    <testcase classname="Inline Input Set Error Exp should render input set menu action button" name="Inline Input Set Error Exp should render input set menu action button" time="0.297" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should not open reconcile dialog on clicking reconcile button, when loading state is true" name="Inline Input Set Error Exp should not open reconcile dialog on clicking reconcile button, when loading state is true" time="0.202" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should open reconcile dialog on clicking reconcile button, when loading state is false &amp; input set is not empty" name="Inline Input Set Error Exp should open reconcile dialog on clicking reconcile button, when loading state is false &amp; input set is not empty" time="0.216" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should open delete input set modal on clicking reconcile button, if input set is empty" name="Inline Input Set Error Exp should open delete input set modal on clicking reconcile button, if input set is empty" time="0.202" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should navigate to input set list view on clicking go back to input set list button" name="Inline Input Set Error Exp should navigate to input set list view on clicking go back to input set list button" time="0.174" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should not delete input set as status is FAILURE" name="Inline Input Set Error Exp should not delete input set as status is FAILURE" time="0.2" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should not delete input set as status is ERROR" name="Inline Input Set Error Exp should not delete input set as status is ERROR" time="0.166" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should not update input set as status is ERROR" name="Inline Input Set Error Exp should not update input set as status is ERROR" time="0.155" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should display the retry button when an error occurs" name="Inline Input Set Error Exp should display the retry button when an error occurs" time="0.206" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Inline Input Set Error Exp should not update input set, when loading state is true" name="Inline Input Set Error Exp should not update input set, when loading state is true" time="0.16" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Old Git Sync Input Set Error Exp should render input set menu action button" name="Old Git Sync Input Set Error Exp should render input set menu action button" time="0.091" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Old Git Sync Input Set Error Exp should open reconcile dialog on clicking reconcile button, when loading state is false &amp; input set is not empty" name="Old Git Sync Input Set Error Exp should open reconcile dialog on clicking reconcile button, when loading state is false &amp; input set is not empty" time="0.248" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Old Git Sync Input Set Error Exp should open delete input set modal on clicking reconcile button, if input set is empty" name="Old Git Sync Input Set Error Exp should open delete input set modal on clicking reconcile button, if input set is empty" time="0.191" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Old Git Sync Input Set Error Exp should navigate to input set list view on clicking go back to input set list button" name="Old Git Sync Input Set Error Exp should navigate to input set list view on clicking go back to input set list button" time="0.163" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Remote Git Sync Input Set Error Exp should render input set menu action button" name="Remote Git Sync Input Set Error Exp should render input set menu action button" time="0.077" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Remote Git Sync Input Set Error Exp should open reconcile dialog on clicking reconcile button, when loading state is false &amp; input set is not empty" name="Remote Git Sync Input Set Error Exp should open reconcile dialog on clicking reconcile button, when loading state is false &amp; input set is not empty" time="0.203" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Remote Git Sync Input Set Error Exp should open delete input set modal on clicking reconcile button, if input set is empty" name="Remote Git Sync Input Set Error Exp should open delete input set modal on clicking reconcile button, if input set is empty" time="0.137" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
    <testcase classname="Remote Git Sync Input Set Error Exp should navigate to input set list view on clicking go back to input set list button" name="Remote Git Sync Input Set Error Exp should navigate to input set list view on clicking go back to input set list button" time="0.137" file="src/modules/70-pipeline/components/InputSetErrorHandling/__tests__/InputSetOutOfSyncYaml.test.tsx">
    </testcase>
  </testsuite>`

Comment (anurag-harness) [reply package.json:277]: Yes, sent it on the channel

**Comment by anurag-harness** [comment]
retrigger jest

**Review by dgarg-harness — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**